src/js/components/FormGroupComponent.jsx

```diff
@@ -33,7 +35,7 @@ var FormGroupComponent = React.createClass({
 
     return (
       <div className="help-block">
-        <strong>{props.errorMessage}</strong>
+        <AutolinkComponent text={props.errorMessage} />
```

Comment:
Only an idea I have, how about writing it like this?
```JSX
<AutolinkComponent>{props.errorMessage}</AutolinkComponent>
```
Looks somehow more intuitive to me.
Accessible in the component by `this.props.children`.
Feel free to take it or not.